config.json

```diff
@@ -1441,6 +1441,18 @@
       "difficulty": 0,
       "topics": null,
       "deprecated": true
+    },
+    {
+      "slug": "grains",
+      "uuid": "5ee66f39-5e37-4907-a6d9-f55d38324c6c",
+      "core": false,
+      "unlocked_by": "hello-world",
+      "difficulty": 1,
```

Comment:
According to the [policies](https://github.com/exercism/java/blob/master/POLICIES.md#reference-tutorial-in-the-first-exercises), you need to reference the hello world tutorial.